locales-src/newtab.ftl

```diff
@@ -51,3 +51,43 @@ newtab-confirm-delete-history-p1 = Are you sure you want to delete every instanc
 newtab-confirm-delete-history-p2 = This action cannot be undone.
 
 newtab-topsites-add-search-engine = Add Search Engine
+
+## Top Site - Action Tooltips.
+# Tooltip to open the context menu for a given page.
+newtab-topsites-tooltip =
+    .title = Open menu
+# Tooltip on an empty topsite box to open the New Top Site dialog.
+newtab-topsites-placeholder-tooltip =
+    .title = Edit this site
+
+## Context Menu: These strings are displayed in a context menu and are meant as a call to action for a given page.
+newtab-menu-edit-topsites = Edit
+newtab-menu-open-new-window = Open in a New Window
+newtab-menu-open-new-private-window = Open in a New Private Window
+newtab-menu-dismiss = Dismiss
+newtab-menu-pin = Pin
+newtab-menu-unpin = Unpin
+newtab-menu-delete-history = Delete from History
+newtab-menu-save-to-pocket = Save to Pocket
+newtab-menu-delete-pocket = Delete from Pocket
+newtab-menu-archive-pocket = Archive in Pocket
+
+## Context Menu - Bookmarks.
+## Note: Bookmark is a noun in "Remove bookmark" but when it is stand alone, just "Bookmark", it is a verb.
+newtab-menu-remove-bookmark = Remove Bookmark
+newtab-menu-bookmark = Bookmark
+
+## Context Menu - Downloaded Menu: In (newtab-menu-copy-download-link, newtab-menu-go-to-download-page), 
+## "Download" in these cases is not a verb, it is a noun. As in, "Copy the link that belongs to this downloaded item".
+newtab-menu-copy-download-link = Copy Download Link
+newtab-menu-go-to-download-page = Go to Download Page
+newtab-menu-remove-download = Remove from History
+
+## Context Menu - Download Menu: These are platform specific strings found in the context menu of an item that has
+## been downloaded. The intention behind "this action" is that it will show where the downloaded file exists on the file
+## system for each operating system.
+newtab-menu-show-file-mac = Show in Finder
+newtab-menu-show-file-windows = Open Containing Folder
+newtab-menu-show-file-linux = Open Containing Folder
+newtab-menu-show-file-default = Show File
```

Comment:
Fluent provides a way to select platform:
https://firefox-source-docs.mozilla.org/intl/l10n/l10n/fluent_tutorial.html#fluent-translation-list-ftl